packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -104,6 +104,37 @@ function apexLanguageServerDatabase(): string | undefined {
   return apexLangServerDbPath;
 }
 
+function lwcTestResults(expectedCwd: string): string {
+  // todo: should this use getRootWorkspacePath instead?
+  const apexDirPath = path.join(
+    expectedCwd,
+    Global.STATE_FOLDER,
+    'tools',
+    'testresults',
+    'lwc'
+  );
+  return apexDirPath;
+}
+
+function testResults(vscodePath: string): string {
+  // todo: should this use getRootWorkspacePath instead?
+  const testResultsDirPath = path.join(
+    vscodePath,
+    Global.STATE_FOLDER,
+    'tools',
+    'testresults'
+  );
+  return testResultsDirPath;
+}
+
+function getApexLanguageServerDatabasePath(): string | undefined {
```

Comment:
kinda think we should throw an error here instead of returning undefined. This method doesn't seem like the right place to verify there is a workspaceFolder (that should be done by the caller I'd think).  We should never hit this code if there is no project open b/c the apex extension should never load I think.